packages/reporter/src/reporter.ts

```diff
@@ -11,7 +11,7 @@ import { type ProcessedFile, type Report, type ReportFormatter, type ReportItem
 export class Reporter {
   readonly basePath: string;
 
-  private report: Report;
+  public report: Report;
```

Comment:
Don't do this public just for test :(

Reply:
yeah i was going back and forth on this. I think its worse doing the type cast in tests. it really doesnt matter that this is public